tumblr-photo-video-ripper.py

```diff
@@ -68,8 +68,27 @@ def run(self):
     def download(self, medium_type, post, target_folder):
         try:
             medium_url = self._handle_medium_url(medium_type, post)
+            medium_url_bak = medium_url
+            medium_url =re.sub(u'[^/]*media.tumblr.com', u'data.tumblr.com', medium_url)
+            if (b'_100.' in medium_url):
```

Comment:
I don't like this exhaustive way. Hard coded is not a good choice.

Why not splitting the string and replacing with `raw` instead?

Reply:
And you should not change at here. Only photos/images are applicable with raw. 

Method `_download(**)` is the right place.